src/components/EarnDetails/EarnDetailsRisks.tsx

```
@@ -0,0 +1,108 @@
+          <ExternalLink href={protocol.url}>
+            {linkText} <OpenInNew />
+          </ExternalLink>
+        )}
+      </Stack>
+      <EarnRiskTagsContainer>
+        <Box sx={(theme) => ({ marginBottom: theme.spacing(4) })}>
+          {tabOptions.map((tab) => (
+            <EarnDetailsRisksNavButton
+              isActive={selectedTag === tab.value}
+              onClick={() => handleTabChange(tab.value)}
+            >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)